packaging/fetch_gpg_key/main.go

```diff
@@ -0,0 +1,108 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// The application fetches encrypted certificate from GCS bucket,
```

Comment:
This comment should be in the form of:
fetch_gpg_key fetches ...

I don't think the presumbits are running as this should have been caught by golint

Reply:
i confirmed locally that golint does not complain about this.